src/reason_pprint_ast.ml

```diff
@@ -2084,16 +2215,10 @@ let protectIdentifier txt =
 let protectLongIdentifier longPrefix txt =
   makeList ~interleaveComments:false [longPrefix; atom "."; protectIdentifier txt]
 
-let combinedAppItems (regularItems, sugaredItems, delimiter, loc) =
-    (regularItems @ sugaredItems, delimiter, loc)
-
 class printer  ()= object(self:'self)
   val pipe = false
   val semi = false
   (* The test and first branch of ternaries must be guarded *)
-  val inTernary = false
-  method underTernary = {<inTernary=true>}
-  method resetInTernary = {<inTernary=false>}
```

Comment:
Thanks to the unparsing pseudo-automated rules, these are no longer needed. We could do the same thing for pipe/semi I imagine.